src/datachain/node.py

```diff
@@ -47,7 +47,7 @@ class DirTypeGroup:
 @attrs.define
 class Node:
     sys__id: int = 0
-    sys__rand: int = -1
+    sys__rand: int = 0
```

Comment:
`sys__rand` is uint in Studio at least, negative default value cause an exception.